templates/certification/model-details.html

```diff
@@ -62,52 +62,56 @@ <h1>{{ vendor }} {{ name }}</h1>
-              {% if release.level == "Enabled" %}
-                <p class="u-no-margin--bottom">Pre-installed in some regions with a custom Ubuntu image that takes advantage of the system’s hardware features and may include additional software. Standard images of Ubuntu may not work well, or at all.</p>
-              {% elif release.level == "Certified" %}
-                <p>The {{ vendor }} {{ name }} development board with the components described below has been awarded the status of certified for Ubuntu.</p>
-                <p class="u-no-margin--bottom">
-                  <a href="{{ release.download_url }}" class="p-button--neutral">Download</a>
-                </p>
-              {% endif %}
-              {% if release.kernel %}
-              <h4 class="u-no-margin--bottom">Kernel</h4>
-              <p>{{ release.kernel }}</p>
+            <li class="{% if release_details['releases']|length > 1 %}p-accordion__group{% endif %}">
+              {% if release_details['releases']|length > 1%}
+              <div role="heading" aria-level="3" class="p-accordion__heading">
+                <button type="button" class="p-accordion__tab" style="background-position: top 1.25rem left 1rem;"id="tab{{ loop.index }}" aria-controls="tab{{ loop.index }}-section" aria-expanded="false"><h3 class="u-no-margin--bottom">{{ release.name }}</h3></button>
+              </div>
+              {% else %}
+              <h3>{{ release.name }}</h3>
               {% endif %}
-              {% if release.notes %}
-              <h4 class="u-no-margin--bottom">Notes</h4>
-              {% for note in release.notes %}
-              <p>{{ note.comment }}</p>
-              {% endfor %}
-              {% endif %}
-              <h4>Hardware</h4>
-              <table aria-label="Hardware" class="u-no-margin--bottom">
-                <tbody>
-                {% for hardware_subtitle, values in release_details["components"].items() %}
-                  <tr>
-                    <th colspan="2" class="p-muted-text">{{ hardware_subtitle }}</th>
-                    <td colspan="8">{% for value in values %} 
-                      <p style="margin-bottom: 0.5rem;">{{ value.name }} {{ value.bus }}({{ value.identifier }})</p>
-                      {% endfor %}
-                    </td>
-                  </tr>
-                {% endfor%}
-                </tbody>
-              </table>
-              <p class="u-sv3">
-                <a href="/certification/{{ canonical_id }}/{{ release.version }}">Hardware details&nbsp;&rsaquo;</a>
-              </p>
-            </section>
-          </li>
+                <section class="{% if release_details['releases']|length > 1 %}p-accordion__panel {% endif %}" id="tab{{ loop.index }}-section" aria-hidden="true" aria-labelledby="tab{{ loop.index }}">
```

Comment:
You are setting aria-hidden even when there is one release which will remove it for the document for screen readers and SEO.

Reply:
Ahh yes, missed this apologies. Will create an issue to fix, thanks